package_lambda

```diff
@@ -5,6 +5,6 @@ OUTPUT_ABSOLUTE=${PWD}/https_checker_lambda_${TIMESTAMP}.zip
 
 VENV=$(pipenv --venv)
 pushd ${VENV}/lib/python3.7/site-packages
-zip -r9 ${OUTPUT_ABSOLUTE} ./*
+zip -r9 ${OUTPUT_ABSOLUTE} ./* -x "./test/*" "./package_lambda"
```

Comment:
At this point $PWD is site-packages so if there's something named test or package_lambda there, it's not from our repo, and it might be related to a dependency, so I'd rather not exclude it.

Reply:
Ah, now I understand. Thanks!
Un-did the change in 2c66182.